src/__tests__/nav.test.js

```diff
@@ -0,0 +1,17 @@
+import renderer from "react-test-renderer";
+import { Provider } from "react-redux";
+import Navbar from "../components/Navbar";
+import store from "../store/configureStore";
+
+const MockNav = () => (
+  <Provider store={store}>
+    <Navbar />
+  </Provider>
+);
+
+describe("Navbar component", () => {
+  it("should match the snapshot", () => {
+    const tree = renderer.create(<MockNav />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Kindly double-check on this test case because it's raising the following error in the image down below. Please make sure that your tests are passing and you'll be good to go 😄 🙏 
![npm 9_11_2021 8_41_43 AM](https://user-images.githubusercontent.com/47625407/132939165-9cecede6-7cfd-4ae4-b1f3-aa0bc5110d23.png)

Reply:
Well Done with fixes !!!